### PR TITLE
[release-3.9] Don't call check_available_rpms.yml when containeriazed installation

### PR DESCRIPTION
**Pull request by nak3**
In containerized installation, we do not expect to check RPM packages
in the repository.

This patch stopps call check_available.yml in containerized
installation.

### CODE REVIEW

File: roles/openshift_version/tasks/masters_and_nodes.yml [hunk 1]
```diff
@@ -10,7 +10,9 @@
     # Both versions have the same string representation
     when: rpm_results.results.versions.available_versions.0 != openshift_version
   # block when
-  when: not openshift_is_atomic | bool
```
[the remark this answers is not in the document]
Comment (nak3) [reply roles/openshift_version/tasks/masters_and_nodes.yml]: Thank you. But I assume that this PR needs for RHEL(RPM)/OCP installation with `containerized = true`. (It just installs atomic-openshift-* services as containers, but still RHEL(RPM), not atomic.)

In such case, users do not want playbook to check openshift RPM packages.

[the remark this answers is not in the document]
Comment (vrutkovs) [reply roles/openshift_version/tasks/masters_and_nodes.yml]: Correct, `openshift_is_containerized` counts that in - its set to true if its an Atomic system or has `containerized = true`

[the remark this answers is not in the document]
Comment (nak3) [reply roles/openshift_version/tasks/masters_and_nodes.yml]: Oh, sorry! I think I misunderstood your first suggestion. I removed `not openshift_is_atomic | bool`. Hopefully I did not misunderstand it now.

**Comment by vrutkovs** [comment]
bot, retest this please

**Pull request merged**